packages/web/src/utils/image.ts

```diff
@@ -1,9 +1,9 @@
 // Simple preloader that resolves on image load or on error
-export const preload = async (src: string) => {
+export const preload = async (src: string): Promise<boolean> => {
   return new Promise((resolve) => {
     const i = new Image()
-    i.onload = resolve
-    i.onerror = resolve
+    i.onload = () => resolve(true)
```

Comment:
interesting, was this just not working?

Reply:
this was working, but it needs to return a boolean for where it's used now